issue_template.md

```diff
@@ -0,0 +1,36 @@
+<!--- Provide a general summary in the title above this comment-->
+
+
+<!--- If you are requesting a new feature, tell us how it should work in free form-->
+<!--- If you are reporting a bug, please submit the detailed description using the template below-->
+
```

Comment:
This will encumber the template, "Expected behaviour/Actual Behaviour" block is kind of the same